README.md

```diff
@@ -1,8 +1,9 @@
-# DeFi Swap AI Assistant 
-<img src="https://i.imgur.com/xUkwJiK.png" alt="cover_image"  width="0" />
+# Ref Finance DeFi Swap Agent
```

Comment:
This README format was to fit the templates repo logic to UI display.

The order must be - image, description, demo, deploy, tooling, project walkthrough, and then you can add anything.

Can you please adjust as it was?

Reply:
Ok will update.  Is the image supposed to not show and be width="0" ?  Or should I keep the new img src that is hosted on github cdn instead of imgur?

Reply:
it's fine to use github.
width='0' is just if you don't wnt to display the image in the readme - i would keep the width=0

Reply:
Ahhh, just realized it is for templates.  Setting both back to 0 now.